src/govee_ble/parser.py

```diff
@@ -71,7 +71,7 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         if local_name.startswith("GV"):
             self.set_device_name(service_info.name[2:].replace("_", " "))
 
-        self.set_precision(2)
+        self.set_precision(1)
```

Comment:
This will change it for all Govee devices this library supports.  We should leave it as is here and change it on a per device basis for devices you have confirmed have a lower precision instead